app/mailers/gallery_validation_mailer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class GalleryValidationMailer < ApplicationMailer
+  def post(gallery:, validation_errors:)
+    @gallery = gallery
```

Comment:
`fail` here if `Rails.application.config.x.gallery_validation_mail_to` is not set.